operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -1316,9 +1317,13 @@ minio:
     requests:
       cpu: 100m
       memory: 128Mi
-  secretKey: supersecret
+  rootPassword: supersecret
   # Changed the mc config path to '/tmp' from '/etc' as '/etc' is only writable by root and OpenShift will not permit this.
   configPathmc: "/tmp/minio/mc/"
+  image:
+    repository: quay.io/minio/minio
+    tag: RELEASE.2022-05-08T23-50-31Z
+    pullPolicy: IfNotPresent
```

Comment:
Ugh, right, we can't set it to null here. On the other hand let's not copy information , cause then we have to keep it in sync with the sub-chart. Let's just keep the changelog entry.

```suggestion
```

Reply:
Yeah that's true. I just realised. Now the image will be permanently hardcoded if leave it like this.